module/zfs/arc.c

```diff
@@ -4043,12 +4043,14 @@ arc_evict_state(arc_state_t *state, uint64_t spa, int64_t bytes,
 
 		/*
 		 * Try to reduce pinned dnodes with a floor of arc_dnode_limit.
-		 * Request that 10% of the LRUs be scanned by the superblock
-		 * shrinker.
+		 * Request that zfs_arc_dnode_reduce_percent of the LRUs be
+		 * scanned by the superblock shrinker.
 		 */
-		if (type == ARC_BUFC_DATA && aggsum_compare(&astat_dnode_size,
+		if (aggsum_compare(&astat_dnode_size,
```

Comment:
Along the lines of performance, we are now calling this regardless of whether we're evicting metadata or data. Can you comment on the performance impact to overall eviction?